core/Tracker/Settings.php

```diff
@@ -110,7 +108,7 @@ protected function getConfigHash(Request $request, $os, $browserName, $browserVe
               $os
             . $browserName . $browserVersion
             . $plugin_Flash . $plugin_Java . $plugin_Director . $plugin_Quicktime . $plugin_RealPlayer . $plugin_PDF
-            . $plugin_WindowsMedia . $plugin_Gears . $plugin_Silverlight . $plugin_Cookie
+            . $plugin_WindowsMedia . $plugin_Silverlight . $plugin_Cookie
```

Comment:
As discussed we should use a `0` as default here instead, to avoid changes in config hashes
```suggestion
            . $plugin_WindowsMedia . '0' . $plugin_Silverlight . $plugin_Cookie
```